src/libraries/Common/tests/TestUtilities/System/WindowsIdentityFixture.cs

```diff
@@ -135,6 +141,73 @@ internal struct USER_INFO_1
             public string usri1_comment;
             public uint usri1_flags;
             public string usri1_script_path;
+
+#if NET7_0_OR_GREATER
+            [CustomMarshaller(typeof(USER_INFO_1), MarshalMode.Default, typeof(USER_INFO_1Marshaller))]
+            public static class USER_INFO_1Marshaller
+            {
+                public static USER_INFO_1Native ConvertToUnmanaged(USER_INFO_1 managed) => new(managed);
+                public static USER_INFO_1 ConvertToManaged(USER_INFO_1Native native) => native.ToManaged();
+                public static void Free(USER_INFO_1Native native) => native.FreeNative();
+
+                [StructLayout(LayoutKind.Explicit)]
```

Comment:
```suggestion
                [StructLayout(LayoutKind.Sequential)]
```

`IntPtr.Size` is 4 on x86, so we can't use explicit layout,

We should ensure the struct layout is correct on 32 bit systems.

Reply:
thanks for spotting this ;)